app/views/hyrax/base/_work_contact_form.html.erb

```diff
@@ -0,0 +1,50 @@
+<% provide :page_title, I18n.t('hyrax.contact_form.title') %>
+
+<!--
+<div class="alert alert-info">
+  <%#= render 'directions' %>
+</div>
+-->
+<h1>
+    <%= t('hyrax.contact_form.header') %>
+</h1>
+
+<% if user_signed_in? %>
+  <% nm = current_user.name %>
+  <% em = current_user.email %>
+<% else %>
+  <% nm = '' %>
+  <% em = '' %>
+<% end %>
+<%= form_for Hyrax::ContactForm.new({message: Rails.root.join('/concern/generics/', @presenter.id)}), url: hyrax.contact_form_index_path,
```

Comment:
Try `request.url` to get the full path and not hardcoded for generic.
Also, lets put it in `subject` and hide that field here. The user can use the category dropdown to tell use what they're asking about